### ISSUE TITLE
Undistinguished DeltaBot Comment

### ISSUE THREAD

**Issue by Snorrrlax** [MystK/delta-bot-three#205]
Link:

https://www.reddit.com/r/changemyview/comments/6vlynb/cmv_love_you_means_the_same_thing_as_i_love_you/dm1dc77/

Also failed to add it to the DeltaLog post:

https://www.reddit.com/r/DeltaLog/comments/6vmkft/deltas_awarded_in_cmv_love_you_means_the_same/

**Comment by wyantb** [comment]
Yeah, there's no retry logic associated with any of the deltalog functionality. Should get #207 in first before worry about this ticket
